wordpress-mobile/WordPress-Android
# Stats revamp v2 views and visitros graph on detail screen

This PR updates Views and Visitors card on detail screen to reflect date selection

Fixes an outstanding item from 16684 

Review/ Merge Instructions

1. [Review corresponding FluxC PR.](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2447)
2. Make sure the corresponding FluxC PR is merged to trunk.
3. Replace FluxC trunk-<hash> in build.gradle.
4. Remove Not Ready for Merge label.
5. Merge the PR.


To test:

Setup:
1. Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
2. Select Debug Settings
3. Find StatsRevampV2FeatureConfig under Features in development enable it and restart app
4. Launch / Re-Launch app
5. Go to Stats either using quick links or menu
6. Switch to Insights tab if necessary
7. Make sure Views & Visitors card is visible on Insights tab. If not present then add it through stats management using either [+ add new stats card ] at the bottom or clicking ⚙️ icon at the top right hand corner.

Test:
- Click on View More on Views card.
- Ensure it takes to respective detail screen
- Ensure a date selector is visible at the top 
- Use < and > arrows on date selectors to move back and forth
- Ensure the cards on the screen reflect the date range selected

NOTE: 
- It may go into loading when navigated to detail screen due to caching 
- the date selector has a  week range as per Week tab

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Updated existing tests

PR submission checklist:
